joplin/base/models.py

```diff
@@ -55,7 +56,39 @@ class HomePage(Page):
     image = models.ForeignKey(TranslatedImage, null=True, on_delete=models.SET_NULL, related_name='+')
 
 
-class JanisPage:
+class JanisPage(Page):
+    parent_page_types = ['base.HomePage']
+    subpage_types = []
+    search_fields = Page.search_fields + [
+        index.RelatedFields('owner', [
+            index.SearchField('last_name', partial_match=True),
+            index.FilterField('last_name'),
+        ])
+    ]
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+    topic = models.ForeignKey(
+        'base.Topic',
+        on_delete=models.PROTECT,
+        verbose_name='Select a Topic',
+    )
+    content_panels = [
+        FieldPanel('topic'),
+        FieldPanel('title'),
+    ]
+
+    @cached_classmethod
```

Comment:
I don't really follow whats happening here. Is this all just to enable the Tabs?

Reply:
Oh, don't know how I missed this comment. 
Yep, so lines 82 & 83 is to allow for children models of JanisPage to define the their own TabbedInterface pieces.
lines 85-90 are to create a default tabbed interface for creating/editing content -- where title and topic are included default fields.

Reply:
I'll try to simplify this some more.